virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -24,48 +32,32 @@ The network attachment definition must be in the same namespace as the pod or vi
 
 [NOTE]
 ====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
+There are multiple methods for configuring a VLAN, for example, creating a node network configuration policy. However, a network attachment definition provides a more efficient and manageable configuration
 ====
 
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
-
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
-
-include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+2]
+include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+1]
```

Comment:
Now I see why you used "Creating a Linux bridge". I think it's a misleading title. I would have called the module "Creating a Linux bridge node network configuration policy" because that's what the user is doing. Oh, well, you can decide whether it's worth changing the title.

Remember that all our nmstate modules are going to be taken over by OCP core, so check xrefs. They should point to the networking assemblies, not our nmstate assemblies.

Reply:
Changed the title of this module to
Creating a Linux bridge node network configuration policy

**Let's go through the xrefs that need to change when we do your office hours on Wednesday, May 11. it's an easy fix but I want to make sure what should change exactly before i make xref changes. Thanks.**

Reply:
Fine iwth me.

Reply:
When we met on May 11, we determined we only needed to get rid of the Xrefs in the virt-using-the-default-pod-network-with-virt assembly. Got rid of those.